src/Raven.Embedded/Raven.Embedded.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>RavenDB.Embedded</id>
+    <version>1.0.0</version>
+    <title>RavenDB Embedded</title>
+    <authors>Hibernating Rhinos</authors>
+    <owners>Hibernating Rhinos</owners>
+    <licenseUrl>http://www.ravendb.net/licensing</licenseUrl>
+    <projectUrl>https://ravendb.net/</projectUrl>
+    <iconUrl>http://static.ravendb.net/logo-for-nuget.png</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>RavenDB Embedded library to run ravendb in embedded way</description>
+    <tags>ravendb embedded database nosql doc db</tags>
+    <repository type="git" url="https://github.com/ravendb/ravendb" />
+    <dependencies>
+      <group targetFramework=".NETStandard2.0">
+        <dependency id="RavenDB.Client" version="4.0.5" exclude="Build,Analyzers" />
+      </group>
+    </dependencies>
+    <contentFiles>
+      <files include="**\RavenDBServer\**\*.*" buildAction="Content" copyToOutput="true"/>
```

Comment:
I'm wondering if we can set 'None' here?

Reply:
None wouldn't add it, would it?
And we need it to be copied to the destination